integrated_channels/utils.py

```diff
@@ -161,7 +161,7 @@ def get_image_url(content_metadata_item):
 
 def is_already_transmitted(transmission, enterprise_enrollment_id, grade):
     """
-    Returns: Boolean indicating if completion date for given enrollment is already sent of not.
+    Return: Boolean indicating if completion date for given enrollment is already sent of not.
```

Comment:
The original was correct for Google/NumPy style docstrings: https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html .  We may need to customize the pydocstyle rules to allow this; [example](https://github.com/edx/cookiecutter-django-app/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/tox.ini#L11-L28)